docs/code/MlNetCookBook.md

```diff
@@ -982,4 +982,4 @@ dynamicPipe = dynamicPipe.Append(new KeyToValueEstimator(env, "PredictedLabel"))
 var dynamicModel = dynamicPipe.Fit(data.AsDynamic);
 
 // Now 'dynamicModel', and 'model.AsDynamic' are equivalent.
-```
\ No newline at end of file
+```
```

Comment:
>`` [](start = 1, length = 2)

nitpicking: please remove the trailing whitespace :)

Reply:
@Zruty0 fixed both :). Seems that GitHub web interface adds a blank line at the end on a commit.

Reply:
Oh I see. Well, I suppose we'll end up with these anyway, if that's what automation is doing. In any case, thanks!

---
In reply to: [222460113](https://github.com/dotnet/machinelearning/pull/1139#discussion_r222460113) [](ancestors = 222460113)